go/exocom/src/client_registry/client_registry.go

```diff
@@ -0,0 +1,77 @@
+package clientRegistry
+
+import "encoding/json"
+
+// Client is the combination of a client name, service type and internal namespace
+type Client struct {
+	ClientName        string
+	ServiceType       string
+	InternalNamespace string
+}
+
+// Route is an entry in the routes
+type Route struct {
```

Comment:
Shouldn't this be in some sort of subsystem, for example the broadcast subsystem?